plan/travisci.go

```
@@ -0,0 +1,72 @@
+package plan
+
+import (
+	"path"
+
+	"github.com/chanzuckerberg/fogg/config"
+	"github.com/chanzuckerberg/fogg/util"
+)
+
+type TravisCI struct {
+	Enabled          bool
+	AWSIDAccountName string
+	AWSProfiles      []AWSProfile
+	TestBuckets      [][]string
+}
+
+func (p *Plan) buildTravisCI(c *config.Config) TravisCI {
+	if p.Accounts == nil {
+		panic("buildTravisCI must be run after buildAccounts")
```

why not return an error instead?